gapis/gfxapi/templates/api.go.tmpl

```diff
@@ -1081,6 +1081,9 @@
     {{range $g := $.Globals}}
       {{Title $g.Name}} {{Template "Go.Type" $g}}
     {{end}}
+    SubcommandIndex gfxapi.SubcommandIndex `nobox:"true"`
```

Comment:
I'm sure I started a comment on this, but I can't find it, so I don't know what your response was.

Instead of polluting all the APIs with mostly Vulkan specific fields, how about we add a single `customState` field / type that can be populated with whatever each API wants?